paper.tex

```diff
@@ -238,7 +238,7 @@ \section*{Abstract}
 
 Lessons take significant effort to build,
 and even more effort to maintain.
-The collaborative code development methods pioneered by the open source community
+The collaborative code development methods pioneered by the open-source community
```

Comment:
I agree that multi-word adjectives are usually hyphenated, but I've never seen "open-source community" - reference?